index.html

```diff
@@ -7,9 +7,21 @@
     <link rel="stylesheet" href="style.css">
     <title>Facebook - log in or sign up</title>
 </head>
-<body>
-    <div>
-        Hello World
-    </div>
+
+<body class="bg-gray-100">
+    <div class="container mt-48 flex mx-auto items-center justify-center">
```

Comment:
Please🙏 use semantic tags instead of div like header or section. 😊

Reply:
- [ ] It is important that for every comment you add you should as well go deep by explaining the `why` a change is being requested. This would assist the students to better understand the reason for a change and they know how best to go about it in their next projects :+1: :sparkles: